conda_smithy/feedstock_content/ci_support/checkout_merge_commit.sh

```diff
@@ -23,6 +23,5 @@ fi
 # Check for merge conflicts.
 if [[ -n "${CIRCLE_PR_NUMBER}" ]]
 then
-    git checkout -qf "pr/${CIRCLE_PR_NUMBER}/merge"
```

Comment:
The other one is on this [line]( https://github.com/conda-forge/conda-smithy/pull/315/files#diff-638180e70af586005a9dcdc586f9bd47R20 ).